cdk/BackendStack.ts

```diff
@@ -183,11 +249,57 @@ export class BackendStack extends Stack {
 				reservedConcurrentExecutions: 10,
 				environment: {
 					CACHE_TABLE_NAME: simDetailsCacheTable.tableName,
+					TABLE_INFO: table.ref,
 				},
+				timeout: Duration.seconds(60),
+				initialPolicy: [
+					new IAM.PolicyStatement({
+						actions: ['timestream:WriteRecords'],
+						resources: [table.attrArn],
+					}),
+					new IAM.PolicyStatement({
+						actions: ['timestream:DescribeEndpoints'],
+						resources: ['*'],
+					}),
+				],
 			},
 		)
-		simDetailsCacheTable.grantWriteData(storeSimInformationOnomondo.fn)
+		simDetailsCacheTable.grantReadWriteData(storeSimInformationOnomondo.fn)
 
+		const dailyOnomondoUpdate = new PackedLambdaFn(
+			this,
+			'dailyOnomondoUpdate',
+			lambdaSources.dailyOnomondoUpdate,
+			{
+				layers: [baseLayer],
+				environment: {
+					CACHE_TABLE_NAME: simDetailsCacheTable.tableName,
+					TABLE_INFO: table.ref,
+				},
+				timeout: Duration.seconds(60),
+				initialPolicy: [
+					new IAM.PolicyStatement({
+						actions: ['timestream:DescribeEndpoints'],
+						resources: ['*'],
+					}),
+					new IAM.PolicyStatement({
+						actions: ['timestream:WriteRecords'],
+						resources: [table.attrArn],
+					}),
+				],
+			},
+		)
+		simDetailsCacheTable.grantReadWriteData(dailyOnomondoUpdate.fn)
+		const dailyRule = new Events.Rule(this, 'InvokeActivitiesDailyRule', {
+			schedule: Events.Schedule.expression('cron(0 0 ? * * *)'),
```

Comment:
Use same type of expression as above.
```suggestion
			schedule: Events.Schedule.expression('rate(1 days)'),
```